cypress/support/commands.js

```diff
@@ -23,3 +23,8 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+// na wypadek nieobsługiwanego obiektu
+// Cypress.on('uncaught:exception', () => {
+//     return false
+//  });
```

Comment:
The only thing you added to this file is comments. Do not merge things like this.